maintainers/scripts/pluginupdate.py

```diff
@@ -559,6 +657,7 @@ def run(
         parser = self.create_parser()
         args = parser.parse_args()
         command = args.command or "update"
+        logging.basicConfig()
```

Comment:
Without this line, any logging using `log` variable didn't work. Also, why default log level is warning? It is not annoying on info level, should I change it too?

Reply:
Logging at info level is a good idea.